fixtures/integrations/jira/stubs/createmeta_response.json

```diff
@@ -49,6 +49,22 @@
               "hasDefaultValue": false,
               "operations": ["set"]
             },
+            "priority": {
```

Comment:
We need to add this to the GET create issue metadata call so that we actually call the new search priorities function in our tests. Jira Server does not call GET create issue metadata so this change only affects the Jira Cloud tests